### ISSUE TITLE
Show and hide buttons have black outline on stage, even though they don't do anything

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-gui#5856]
### Expected Behavior
Disabled buttons should not react when clicked

### Actual Behavior
Show and hide buttons in the target pane don't do anything on the stage, but it toggles a black outline on the buttons that make it seem like it is doing something

### Steps to Reproduce
Click on backdrops, then press these buttons:
<img width="149" alt="Screen Shot 2020-06-04 at 14 36 32" src="https://user-images.githubusercontent.com/2855464/83799342-a6e9a000-a673-11ea-94f4-c961c980c9a8.png">

### Operating System and Browser
all

**Comment by fsih** [comment]
help wanted candidate?

**Comment by apple502j** [comment]
Duplicate of #1517
(I remember most of the issues I created :P)

**Comment by BoomerScratch** [comment]
It seems to be the focus, so clicking somewhere else will remove the black outline.
![](https://user-images.githubusercontent.com/60622217/83881664-46289900-a741-11ea-9b10-432522f96891.gif)
Or is that the thing that needs to be fixed?

**Comment by redstone2010** [comment]
> It seems to be the focus, so clicking somewhere else will remove the black outline.
> ![](https://user-images.githubusercontent.com/60622217/83881664-46289900-a741-11ea-9b10-432522f96891.gif)

True...

**Comment by Wowfunhappy** [comment]
I believe this has been fixed as of the current live version of Scratch?

**Comment by mxmou** [comment]
> I believe this has been fixed as of the current live version of Scratch?

Yes, it was fixed together with the high contrast changes. I'll close the linked PR.
